http-clients/apache-client/src/main/java/software/amazon/awssdk/http/apache/ApacheHttpClient.java

```diff
@@ -559,6 +564,16 @@ public void setCredentialsProvider(CredentialsProvider credentialsProvider) {
             credentialsProvider(credentialsProvider);
         }
 
+        @Override
+        public Builder tcpKeepAlive(Boolean keepConnectionAlive) {
+            standardOptions.put(SdkHttpConfigurationOption.TCP_KEEPALIVE, keepConnectionAlive);
+            return this;
+        }
+
+        public void setTcpKeepalive(Boolean keepConnectionAlive) {
```

Comment:
It seems the setter also needs to be updated to `setTcpKeepAlive`

Reply:
Missed that one, thank you.